cloud/passportauth/integration/UserDataRepository.ts

```diff
@@ -0,0 +1,36 @@
+import * as Promise from 'bluebird';
+import * as _ from 'lodash';
+import {User} from './UserSeedData';
+
+export interface DataRepository<T> {
```

Comment:
**Sorry for repeating the same comments but previous comment wasn't taken into account.**

Drop generic and add User as prefix to name. You building your own case for user. 
You have only one object that needs to be fetched from db. 
We can have RoleRepository - but I think that it's overcomplicating things. Just stick with user.